individual_project.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<link rel="stylesheet" href="assets/styles/style.css" />
+	</head>
+
+	<body class="baa">
+        <div class="home-nav navigation">
+              <div class="logo">
+                 <a href="index.html"><img src="assets/img/logo.png" alt="JumboCode Logo"></a>
+              </div>
+             <ul class="nav-links">
+                <li><a href="about.html">About Us</a></li>
+                <li><a href="projects.html">Projects</a></li>
+                <li><a href="join.html">Join</a></li>
+                <li><a href="contact.html">Contact</a></li>
+            </ul>
+        </div>
+        <br />
+        <br />
+        <div class="organization quote-right background"> <!-- why won't the line stay in line with the text-->
+            <h1>Boston<br />Athletics<br />Association</h1>
+        </div>
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+        <br /> <!--there's probably a more elegant way to do this?-->
+        
+	    <div class="content">
+		    <div class="screenshot">
+		    	<img src="iphone_mockup.png" width="500" height="500"> 
+		    	<!--I guess we need to add in the screenshot of the website too-->
+		    </div>
+		    <div class="client_info">
+		    	<!--I'm not very good at making repeatable classes, I'm sorry. Please help-->
```

Comment:
don't sell yourself short - these class names are very general, which is great. I think generally we use hyphens rather than underscores in class names, so just change that and that should be good.